Sources/PackageModel/SwiftSDK.swift

```diff
@@ -509,15 +527,29 @@ public struct SwiftSDK: Equatable {
         )
     }
 
-    /// Returns `macosx` sdk platform framework path.
+    /// Auxiliary platform frameworks and libraries.
+    ///
+    /// The referenced directories may contain, for example, test support utilities.
+    ///
+    /// - SeeAlso: ``sdkPlatformFrameworkPaths(for:environment:)``
+    public struct PlatformPaths {
+        /// Path to the directory containing auxiliary platform frameworks.
+        public var frameworks: AbsolutePath
+
+        /// Path to the directory containing auxiliary platform libraries.
+        public var libraries: AbsolutePath
+    }
+
+    /// Returns framework paths for the provided Darwin platform.
     public static func sdkPlatformFrameworkPaths(
+        for darwinPlatform: DarwinPlatform,
         environment: EnvironmentVariables = .process()
-    ) throws -> (fwk: AbsolutePath, lib: AbsolutePath) {
```

Comment:
We'll have to keep the tuple version of the function for a bit so that clients of libSwiftPM don't immediately break. But the old version returning a tuple should be deprecated.